adafruit_ads1x15/ads1x15.py

```diff
@@ -75,6 +83,7 @@ def __init__(
         gain: float = 1,
         data_rate: Optional[int] = None,
         mode: int = Mode.SINGLE,
+        compqueue: int = 0,
```

Comment:
Please document this.

Reply:
Comments added for this addition as well as new comparator_low_thres and comparator_high_thres properties